engine/src/main/scala/cromwell/engine/workflow/package.scala

```diff
@@ -0,0 +1,40 @@
+package cromwell.engine
+
+import akka.actor.{ActorSystem, ActorRef}
+import com.typesafe.scalalogging.StrictLogging
+import cromwell.services.MetadataServiceActor.{MetadataServiceAction, PutMetadataAction, MetadataServiceResponse, MetadataEvent}
+
+import scala.concurrent.{Future, ExecutionContext}
+
+package object workflow {
+  // Sent when there is a failure to push to the metadata service
+  private[workflow] case class MetadataServiceFailureResponse(reason: Throwable, action: Option[MetadataServiceAction] = None)
```

Comment:
Mind if I move all this stuff to something like a MetadataUtils? Or any other place? It seems like this package object is Metadata heavy currently.